app/src/main/res/values/strings.xml

```diff
@@ -100,6 +98,7 @@
     <string name="notification_channel_description">Exposure notification alerts.</string>
     <string name="menu">Menu</string>
     <string name="powered_by">Powered by</string>
+    <string name="powered_by_logo_content_description">Covidwatch</string>
```

Comment:
I think it will be better if we use real name spelling meaning "Covid Watch". This way text-to-speech algorithm will generate more appropriate speech.

Reply:
I spend too much time looking at the repo and package name ;)